disco_aws_automation/disco_storage.py

```diff
@@ -195,16 +192,6 @@ def configure_storage(self,
         logging.debug("mapped %s to root partition", disk_names[current_disk])
         current_disk += 1
 
-        # Map the latest snapshot for this hostclass
-        if map_snapshot:
-            latest = self.get_latest_snapshot(hostclass)
-            if latest:
-                self.wait_for_snapshot(latest)
-                current_name = disk_names[current_disk]
-                bdm[current_name] = self.create_snapshot_bdm(latest, iops)
-                logging.debug("mapped %s to snapshot %s", current_name, latest.id)
-                current_disk += 1
-
         # Map extra disk
         if extra_disk:
```

Comment:
@pchand-amplify , @jichen-amplify  maybe the right solution to the problem is using lambda on the hook. so maybe treat lambda function as our S3 bucket as account specific artifact that provide magic snapshot function is the right way. and it won't complicate the existing code and usages